src/io/flutter/ProjectOpenActivity.java

```diff
@@ -34,7 +34,9 @@
   public static final ProjectType FLUTTER_PROJECT_TYPE = new ProjectType("io.flutter");
   private static final Logger LOG = Logger.getInstance(ProjectOpenActivity.class);
 
-  @NotNull private final FlutterSettings settings;
+  private FlutterSettings settings;
```

Comment:
Perhaps mark this as `@Nullable`?

Reply:
Done. Compiling with the latest Dart plugin would have caught the other one of these problems.

Reply:
And undone, since it caused the build to go red.